src/citrine/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '1.31.1'
+__version__ = '1.32.1'
```

Comment:
```suggestion
__version__ = '1.31.2'
```
This should either be version `1.32.0` or `1.31.2`.  Probably 1.31.2, since this is just a documentation change.